string ExecutionState
float64 alarmLatitude
float64 alarmLongitude